service/docs/source/overview.rst

```diff
@@ -797,6 +797,8 @@ generate a report:
    directly to ``libgeopm.so.2`` at compile time.
 3. The ``GEOPM_REPORT`` environment variable must be set in the
    environment of the ``geopmctl`` process.
+4. The ``GEOPM_CTL_LOCAL`` environment variable must be set which generates
```

Comment:
Minor recommendation: maybe just add a note explaining why this option is needed in the non-MPI case.